docs/user/dashboard/dashboard.asciidoc

```diff
@@ -0,0 +1,472 @@
+[[dashboard]]
+= Dashboard
+
+[partintro]
+--
+
+A _dashboard_ is a collection of panels that you use to analyze your data. On a dashboard, you can add a variety of panels that 
+you can rearrange and tell a story about your data. Panels contain everything you need, including visualizations, 
```

Comment:
Is `everything you need` claiming too much?

Reply:
Maybe. Any suggestions?